vhost-device-sound/src/audio_backends.rs

```diff
@@ -90,8 +99,13 @@ mod tests {
             let value = try_backoff(|| alloc_audio_backend(v, Default::default()), std::num::NonZeroU32::new(3)).expect("reached maximum retry count");
             assert_eq!(TypeId::of::<PwBackend>(), value.as_any().type_id());
         }
-        #[cfg(all(feature = "alsa-backend", target_env = "gnu"))]
-        {
+    }
+
+    #[cfg(all(feature = "alsa-backend", target_env = "gnu"))]
+    rusty_fork_test! {
```

Comment:
I'd add a comment here and in the other tests cases mentioning why we are using `rusty_fork_test!` so in the future we will remember to use it for other tests where using env.

WDYT?

Reply:
Excellent idea, will do!

Reply:
Annotated all rusty_fork_test locations with comments